### ISSUE TITLE
Setup package releases & CI/CD flow

### ISSUE THREAD

**Issue by dermidgen** [mysticcoders/mysticrobots#60]
Initial CI/CD flow:

 * GitFlow; branch-based development
 * PR on branch -> master
 * Changes to master always fire unit/integration tests; by package
 * Tags on master merge to staging on test pass
 * Changes to staging fire up the client & server and run e2e tests
 * Staging merges to production on test pass
 * Production runs migrations and/or deployments

Manual release strategy
 * Write protect on master, staging, production branches
 * Only those with write on master can release
 * Manually release with lerna publish
 * Tags affected packages for release

**Comment by kinabalu** [comment]
yeah, so a PR would get reviewed and approved and can get merged by someone with the appropriate permissions and given above things would merge into staging and then finally production. I like it
